### PR TITLE
Variant List and Vol Pricing UI polishes

### PR DESCRIPTION
**Pull request by sofiamatulis**
**Variant List UI polishes:** https://github.com/Shopify/dawn/issues/2782
**Vol Pricing UI polishes**: https://github.com/Shopify/dawn/issues/2784

This PR includes
- Sold out text shows info icon and hover state. See video [here](https://screenshot.click/07-26-9esnf-ja8a2.mp4). Think we should just remove hover state (background change) when an item is sold out, but still allow for the popover to be accessed (from @scottgmeadows)
- Spacing/inner padding in the volume pricing table (both inline PDP and on the popover) is too large. Current inner padding seems to be padding: 0.8rem 1.2rem;... Can we tweak to padding: 0.6rem 0.8rem; ? (from @scottgmeadows). [Current PR](https://screenshot.click/Current.png) | [Expected Figma](https://screenshot.click/Expected-Figma.png).
- ❗️Alignment issues in centering elements across the variant row with Variant name, Quantity, Price, and Variant total... there are several different cases/permutations with and without SKU's, price per unit (weight), Sold out state, etc. [Figma file reference](https://www.figma.com/file/ov2FHR3uvfVzKqbVV33IPz/Variant-list-on-PDP?type=design&node-id=4643-101322&mode=design).
- "Volume pricing available" note in product grid is wrong colour. [Current PR](https://screenshot.click/06-34-ksy2h-eu7ax.png) | [Expected](https://screenshot.click/06-35-heupf-8nlvy.png). Should match opacity value of volume pricing note on PDP.

Editor: https://os2-demo.myshopify.com/admin/themes/140239437846/editor

### CODE REVIEW

File: assets/component-volume-pricing.css [hunk 1]
```diff
@@ -51,4 +51,4 @@ price-per-item,
 .price-per-item {
   font-size: 1.4rem;
   letter-spacing: 0.06rem;
-}
+}
```
Comment (KaichenWang) [inline assets/component-volume-pricing.css:54]:
```suggestion
}

```

File: snippets/variant-list-row.liquid [hunk 2]
```diff
@@ -127,13 +127,22 @@
...
+      unless variant.available == false or quantity_rule_soldout
+        assign is_available = true
+      endunless
+      if has_qty_rules or has_vol_pricing
+        assign has_popover = true
+      endif
     -%}
     <quantity-popover>
-      <div class="variant-item__quantity-wrapper quantity-popover-wrapper{% if has_qty_rules == false and  has_vol_pricing == false %} variant-item__quantity-wrapper--no-info{% endif %}">
+      <div class="variant-item__quantity-wrapper quantity-popover-wrapper{% if has_qty_rules == false and has_vol_pricing == false %} variant-item__quantity-wrapper--no-info{% endif %}">
```
Comment (KaichenWang) [inline snippets/variant-list-row.liquid:141]: Is this equivalent?
```suggestion
      <div class="variant-item__quantity-wrapper quantity-popover-wrapper{% unless has_popover %} variant-item__quantity-wrapper--no-info{% endunless %}">
```

**Review by KaichenWang — APPROVED** [review]
Code looks good

**Comment by edmund-teh** [comment]
@sofiamatulis 

- [ ] Is it possible to have the `Price` and `Variant total` values line up when SKU's are turned on **AND** there's a unit price? Would we have to build too much logic around this or like have an empty string directly under the `Variant total` value?

Current PR: 
https://screenshot.click/Current-PR-SKUs-Unitprice-1.png
https://screenshot.click/Current-PR-SKUs-Unitprice-2.png

Figma reference: https://screenshot.click/Figma-reference-SKUs-Unitprice.png
[Figma file here](https://www.figma.com/file/ov2FHR3uvfVzKqbVV33IPz/Variant-list-on-PDP?type=design&node-id=4643-101323&mode=design&t=igVKvZPyMgJfHwnM-4)

**Comment by scottgmeadows** [comment]
- Spacing for the PDP volume pricing table is good.

- Spacing for the volume pricing table in the variant list and cart popovers is still too large.

[Video here](https://screenshot.click/07-15-h8mut-4gi5d.mp4)

**Review by eugenekasimov — APPROVED** [review]
👍

**Review by edmund-teh — APPROVED** [review]
The changes looks so much better! Thanks @sofiamatulis

**Pull request merged**